tests/integration/schema_registry/test_SchemaRegistryClient.py

```diff
@@ -0,0 +1,295 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright 2020 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import json
+from uuid import uuid1
+
+from confluent_kafka.schema_registry.schema_registry_client import CompatibilityType
+
+
+def _subject_name(prefix):
+    return prefix + "-" + str(uuid1())
+
+
+def cmp_schema(schema1, schema2):
+    schema_dict1 = json.loads(schema1)
+    schema_dict2 = json.loads(schema2)
+
+    return all({v == schema_dict2[k] for k, v in schema_dict1.items()})
```

Comment:
This will raise an exception if k is not in dict2, right? Maybe not the desired behaviour, but okay for a test I guess.